src/IO/interfile.cxx

```diff
@@ -1262,120 +1262,150 @@ write_basic_interfile_PDFS_header(const string& header_file_name,
     output_header << "!matrix size [" << order_of_z << "] := ";
     // tedious way to print a list of numbers
     {
       std::vector<int>::const_iterator seg = segment_sequence.begin();
       output_header << "{ " <<pdfs.get_proj_data_info_ptr()->get_num_axial_poss(*seg);
       for (seg++; seg != segment_sequence.end(); seg++)
-	output_header << "," << pdfs.get_proj_data_info_ptr()->get_num_axial_poss(*seg);
+    output_header << "," << pdfs.get_proj_data_info_ptr()->get_num_axial_poss(*seg);
       output_header << "}\n";
     }
 
     output_header << "matrix axis label [" << order_of_bin << "] := tangential coordinate\n";
     output_header << "!matrix size [" << order_of_bin << "] := "
-		  <<pdfs.get_proj_data_info_ptr()->get_num_tangential_poss() << "\n";
+          <<pdfs.get_proj_data_info_ptr()->get_num_tangential_poss() << "\n";
+
+    // IF TOF is supported add this in the header.
+    if (pdfs.get_proj_data_info_ptr()->get_scanner_ptr()->is_tof_ready() &&
+            pdfs.get_proj_data_info_ptr()->get_num_tof_poss() > 1)
+    {
+        // Moved in scanner section
+//    output_header << "%number of TOF time bins :=" <<
+//                     pdfs.get_proj_data_info_ptr()->get_scanner_ptr()->get_num_max_of_timing_bins()  << "\n";
+    output_header << "%TOF mashing factor := " <<
+                     pdfs.get_proj_data_info_ptr()->get_tof_mash_factor() << "\n";
+    }
   }
 
-  const  ProjDataInfoCylindrical* proj_data_info_ptr = 
+  const  ProjDataInfoCylindrical* proj_data_info_ptr =
     dynamic_cast< const  ProjDataInfoCylindrical*> (pdfs.get_proj_data_info_ptr());
 
    if (proj_data_info_ptr!=NULL)
      {
        // cylindrical scanners
-   
-       output_header << "minimum ring difference per segment := ";    
+
+       output_header << "minimum ring difference per segment := ";
        {
-	 std::vector<int>::const_iterator seg = segment_sequence.begin();
-	 output_header << "{ " << proj_data_info_ptr->get_min_ring_difference(*seg);
-	 for (seg++; seg != segment_sequence.end(); seg++)
-	   output_header << "," <<proj_data_info_ptr->get_min_ring_difference(*seg);
-	 output_header << "}\n";
+     std::vector<int>::const_iterator seg = segment_sequence.begin();
+     output_header << "{ " << proj_data_info_ptr->get_min_ring_difference(*seg);
+     for (seg++; seg != segment_sequence.end(); seg++)
+       output_header << "," <<proj_data_info_ptr->get_min_ring_difference(*seg);
+     output_header << "}\n";
        }
 
        output_header << "maximum ring difference per segment := ";
        {
-	 std::vector<int>::const_iterator seg = segment_sequence.begin();
-	 output_header << "{ " <<proj_data_info_ptr->get_max_ring_difference(*seg);
-	 for (seg++; seg != segment_sequence.end(); seg++)
-	   output_header << "," <<proj_data_info_ptr->get_max_ring_difference(*seg);
-	 output_header << "}\n";
+     std::vector<int>::const_iterator seg = segment_sequence.begin();
```

Comment:
more white space

Reply:
How would you like me to address this? Revert the file and reply the changes?